Schutzfile

```diff
@@ -46,5 +46,12 @@
         "commit": "35de3093a7b52569512bdc61d2105febbb9b0c7e"
       }
     }
+  },
+  "rhel-9.0": {
+    "dependencies": {
+      "osbuild": {
+        "commit": "152b14a0d6c5deccfce2096308230f6d9067d23e"
```

Comment:
What version do we need? Why? If we are to release this into RHEL9 will we also make sure to coordinate with osbuild release? cc @gicmo

Reply:
Ah, I picked a very random one. osbuild 30 with 234997eeb3e0355fd67373849176a49130f9e0e7 should be enough.